tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers_test.ts

```diff
@@ -93,4 +98,398 @@ describe('Debugger reducers', () => {
       });
     });
   });
+
+  it('Overrides activeRunId on debuggerRunsLoaded', () => {
+    const state = createDebuggerState();
+    const nextState = reducers(
+      state,
+      actions.debuggerRunsLoaded({
+        runs: {
+          __default_debugger_run__: {
+            start_time: 222,
+          },
+        },
+      })
+    );
+    expect(nextState.activeRunId).toEqual('__default_debugger_run__');
+  });
+
+  it('Updates load state on numExecutionsRequested', () => {
+    const state = createDebuggerState({
+      runs: {
+        __default_debugger_run__: {
+          start_time: 111,
+        },
+      },
+      runsLoaded: {
+        state: DataLoadState.LOADED,
+        lastLoadedTimeInMs: 222,
+      },
+      activeRunId: '__default_debugger_run__',
+    });
+    const nextState = reducers(state, actions.numExecutionsRequested());
+    expect(nextState.executions.numExecutionsLoaded.state).toEqual(
```

Comment:
Can you ever go from LOADED to LOADING? A spec would be nice :)

Reply:
Yes. I added a spec in debugger_types.ts.